check_source.py

```diff
@@ -350,6 +350,10 @@ def is_devel_project(self, source_project, target_project):
             return True
 
         # Allow any projects already used as devel projects for other packages.
+        if target_project.endswith('openSUSE:Factory'):
+            devel_pkgs = git_factory_devel_project_dict_get(self.apiurl)
+            return True if source_project in devel_pkgs.values() else False
```

Comment:
the "else False" disables the entire logic afterwards. are we planning to do this today rather than an opt in?

Reply:
This is already a done deal. If OBS cannot store some of the package devel project info correctly, it may as well not store any of the information there anymore.

Furthermore, the attribute that sets devel projects in Factory is turned off for a few weeks as it's not possible to accept git-based new packages to Factory.

Reply:
well, it means that new package additions need to add their devel project here to this project first. did we communicate that?

Reply:
I see... my mistake. This is something else and you are correct, the `else False` is thus wrong here. We still want this attribute...

Reply:
So, I think there's nothing really to add here. New devel projects still need whitelisting before packages can pass this check. The whitelist is above.

For packages that come from devel projects for other packages, this is the change we want. We store the informatino on the "backend" in different place now than the XML of the package, that's all.

For non-openSUSE:Factory packages, this fallsthrough.